### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by JacobEdelman**
Note that this is intended to be a readme for the club-in-a-box repository after it has been renamed to hackedu. It is a very rough draft.
Moved from docs repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,54 @@
...
+- Mention how Github works with emails.
+- Fix spelling, grammar, and general English mistakes
+- Improve the tone/humour
+- Add more to and improve the README Todo
+- Fix the sixth item on the README Todo, its just a bad joke
+- Change step 3 to make it more of a conclusion
+- figure out readme capitalization properly 
+
+
+
```
Comment (MaxWofford) [inline README.md]: Extra lines don't matter much, but we don't need them.

File: README.md [hunk 2]
```diff
@@ -1 +1,54 @@
...
+## Step 1: You have issues, I have issues, we all have issues!
+Every Github project has issues. Literally.
+
+### Issues == Reddit_posts  and Issues == Facebook_posts
+An issues has a topic, some content, and can be commented upon just like a post on many social networks. An issues made for a github project is typically about a bug, recommendation, or relevant discussion for the project. Issues can also have tags and can be search for. Old issues (that have been resolved or are no longer relevant) are closed, which simply means that they no longer appear when searching for open threads. They are still archived and can even be reopened if needed. If you need help anywhere in this tutorial just open up an issue and we'll get right on it. 
+
+### Lets talk with issues
+In [project name] we use issues for any discussion relevant to running a club. If you have a question, proposal, or something you want to say of general interest, just open up an issue!
+
+## Step 2: Contribute!
```
Comment (MaxWofford) [inline README.md]: Do we want this section in `CONTRIBUTING.md`, so github will display it on PR/issue submission pages?

Comment (JacobEdelman) [inline README.md]: Yeah, that would be cool, how would I do that?

Comment (MaxWofford) [inline README.md]: I think anything in `CONTRIBUTING.md` will be shown here. sauce~ https://github.com/blog/1184-contributing-guidelines

Comment (JacobEdelman) [inline README.md]: Hmm... well, On second thought I'd think I'd prefer everything to be in readme since its meant to be one file.

**Comment by zachlatta** [comment]
Slammed right now—I'll try to get to this sometime next week. So sorry for the delay! D:

**Comment by MaxWofford** [comment]
Hey @JacobEdelman. Sorry for the delays on getting this merged. I'd be happy to if you resolve the merge-conflicts. Checkout the `git rebase -i master` command to do this. 

If you have any questions please don't hesitate to ask!

**Comment by zachlatta** [comment]
Urgh - closing for now due to lack of communication. So sorry @JacobEdelman for dropping the ball on getting to this urgently.

Please don't hesitate to reopen.

**Pull request closed without merging**